packages/skeleton/src/lib/components/ListBox/ListBoxItem.svelte

```diff
@@ -17,19 +17,23 @@
 	export let name: string;
 	/** Set the input's value. */
 	export let value: any;
+	/** Disables selection of the item. */
+	export let disabled: boolean = false;
 
 	// Context
+	export let disabledItems: boolean = getContext('disabledItems');
```

Comment:
Instead of introducing a new prop to obtain the context, we can utilize the following approach:

```suggestion
	export let disabled: boolean = getContext('disabled');
```
This way, the disabled status will be inferred from the parent component, while still allowing users the flexibility to override this value for individual listbox items.

Reply:
yes that was the behavior I was looking, got confused somehow, thanks